source/geod24/concurrency.d

```diff
@@ -1057,6 +1111,8 @@ private:
                 pos = 0;
             }
         }
+
+        thisScheduler(null);
```

Comment:
Not in a `scope (exit)` at the beginning? I guess it doesn't matter after an exception is thrown..

Reply:
That should be scoped regardless, just to be safe.